src/style.css

```diff
@@ -1,3 +1,7 @@
+.table-responsive .autocomplete-result-list {
+  position: relative !important;

```

Comment:
Was there no other way than using `!important`?

Reply:
Because the positioning is added dynamically with JavaScript when the list is displayed like this `style=position: absolute; ..."`.

Reply:
And this is from the autocomplete plugin?

Reply:
correct